src/battle_interface.c

```diff
@@ -764,14 +764,23 @@ static void InitLastUsedBallAssets(void)
     gBattleStruct->ballSpriteIds[1] = MAX_SPRITES;
 }
 
+// This function is here to cover a specific case - one player's mon in a 2 vs 1 double battle. In this scenario - display singles layout.
+u32 WhichBattleCoords(u32 battlerId) // 0 - singles, 1 - doubles
+{
+    if (GetBattlerPosition(battlerId) == B_POSITION_PLAYER_LEFT && gPlayerPartyCount == 1)
```

Comment:
I think `gPlayerPartyCount` should be `CountBattleReadyMons`, since the player can enter a 1v2 battle with a full party if only 1 is alive

Reply:
It's fine as it is I think. gPlayerPartyCount also counts fainted mons, so if the player enters a 1v2 with a full party but only 1 is alive, it will treat the battle like a double battle.